pom.xml

```diff
@@ -2518,6 +2517,11 @@
         <artifactId>httpclient</artifactId>
         <version>${httpclient.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.httpcomponents</groupId>
+        <artifactId>httpcore</artifactId>
+        <version>${httpcore.version}</version>
```

Comment:
Do not declare version. Leave it for the dependency management.

Reply:
Had to remove all dependencies (httpClient & HttpCore) since they were at dependencyManagement scope